ezio3.py

```diff
@@ -0,0 +1,79 @@
+import random
+from itertools import cycle
+
+from music import tone, play_sequence, play_drumbase, Scale
+from synth import play_drum2, play_drum3, play_kick
+
+def gen_rythm2(beats, prob=0.9):
+    if beats == 1:
+        return [1]
+    if random.random() < prob:
+        prob = max(prob-.2, .1)
+        return [*gen_rythm2(int(beats/2), prob), *gen_rythm2(int(beats/2), prob)]
+    else:
+        return [beats]
+
+def drill(notes):
+    return [note if (random.random()>.2) else 0 for note in notes]
+
+def drill_pattern(notes, pattern):
+    return [note*p for p, note in zip(cycle(pattern), notes)]
+
+def drumify(rythm):
+    beats = []
+    for d in rythm:
+        beats.append(1)
+        beats.extend([0]*(d-1))
+    return beats
+
+def make_music(synth):
+    MUL = 2
+    scale = Scale('E', 'melodic minor')
```

Comment:
```suggestion
    scale = Scale('C', 'melodic minor')
```
The names of the chords underneath were originally based on C.